src/app/app.component.ts

```diff
@@ -51,6 +56,11 @@ export class AppComponent implements OnInit, OnDestroy {
     }
 
     ngOnInit() {
+        this._nssService.currentItem.subscribe(item => this.item = item);
+        this._nssService.currentRegRegion.subscribe(regRegion => this.regRegion = regRegion);
+        this._nssService.currentStatisticGroup.subscribe(statisticGroup => this.statisticGroup = statisticGroup);
+        this._nssService.currentClone.subscribe(clone => this.clone = clone);
+
```

Comment:
I don't think these are needed since these items aren't used elsewhere in the app component.